ui-tests/src/test/java/io/syndesis/qe/pages/integrations/detail/IntegrationDetailPage.java

```diff
@@ -19,8 +19,8 @@
 	public final List<String> actionsSet = new ArrayList<String>();
 
 	private static final class Element {
-		public static final By ROOT = By.cssSelector("syndesis-integration-detail-page");
-		public static final By STATUS = By.cssSelector("syndesis-integration-status");
+		public static final By ROOT = By.cssSelector("syndesis-integrations-detail-page");
+		public static final By STATUS = By.cssSelector("syndesis-integrations-status");
```

Comment:
Are you sure with these? I'm not sure if we are using same version but i got integration without s.

Reply:
in TP3 it is again with 's', (but not everywhere) see scenario @db-connection-crud-1-read-update comment

Reply:
OK, probably it will be best to wait.

Reply:
I can see singular variant without `s`. I'd rather exclude it from PR for now and pls double-check the deployed version.

Reply:
ok